### PR TITLE
Publish Lens extension howto

### PR DESCRIPTION
**Pull request by smaffulli**

### CODE REVIEW

**Review by AjayTripathy — APPROVED** [review]

File: _posts/2021-06-10-lens-kubecost-extension.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+---
+title: "How to use the Kubecost extension with Lens IDE"
+description: "Kubecost allows Azure AKS administrators to allocate and
```
Comment (dwbrown2) [inline _posts/2021-06-10-lens-kubecost-extension.md:3]: This looks like it's carried over from another article...

File: _posts/2021-06-10-lens-kubecost-extension.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+For most teams, it’s really difficult to understand the cost of Kubernetes workloads in an even somewhat static multi-tenant environment. When concepts like autoscaling are introduced, it becomes far more complex. This creates many challenges, including unintentional overspending, poor visibility for business planning, and missing costly bugs/mistakes 
+
+The [open source integration](https://github.com/kubecost/kubecost-lens-extension) provides a direct view into these costs at different levels so you can understand what is driving overall costs and how efficiently they are being spent. In this first release you can see cost trends for any workload aggregation, e.g. pod, deployment, statefulset, namespace, etc. 
+
+<video src="/assets/images/2021-06-10-lens-kubecost/lens-kubecost-extension.mov" width="860" height="537" controls preload></video>
+
+## Install Kubecost in your Kubernetes cluster using Lens IDE
+
+Get started by installing the Kubecost stack in your cluster. Open Lens and connect to your Kubernetes cluster.  Go to the Apps directory and search for the _cost-analyzer_ chart.
```
Comment (dwbrown2) [inline _posts/2021-06-10-lens-kubecost-extension.md:28]: It's not clear what the exact steps are... do you just click install?

File: _posts/2021-06-10-lens-kubecost-extension.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+
+Create a namespace called “kubecost” and install Kubecost cost-analyzer in it
+
+![Install Kubecost in the kubecost namespace](/assets/images/2021-06-10-lens-kubecost/image5.png "Install Kubecost in the kubecost namespace")
+
+Once Kubecost is installed, you can add the Kubecost extension
+
+
+*   Download the [latest tar.gz file release from GitHub](https://github.com/kubecost/kubecost-lens-extension/releases). Alternatively, just copy the download URL of the release artifact.
```
Comment (dwbrown2) [inline _posts/2021-06-10-lens-kubecost-extension.md:40]: @AjayTripathy this seems very raw... is this actually the best way to install?

File: _posts/2021-06-10-lens-kubecost-extension.md (hunk 2 above)
Comment (smaffulli) [reply _posts/2021-06-10-lens-kubecost-extension.md:28]: Yes. The screenshot below should make it clear.

File: _posts/2021-06-10-lens-kubecost-extension.md (hunk 3 above)
Comment (smaffulli) [reply _posts/2021-06-10-lens-kubecost-extension.md:40]: it is raw but it's what all extensions need to use. 
image

File: _posts/2021-06-10-lens-kubecost-extension.md (hunk 1 above)
Comment (smaffulli) [reply _posts/2021-06-10-lens-kubecost-extension.md]: fixed in new commit.

File: _posts/2021-06-10-lens-kubecost-extension.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+This will add a new panel in Lens UI showing costs of resources per namespace. 
+
+![See the new Costs panel in the Lens UI](/assets/images/2021-06-10-lens-kubecost/image6.png "See the new Costs panel in the Lens UI")
+
+
+To see the full Kubecost UI, from the Lens terminal, setup a port forward to your Kubernetes cluster with kubectl:
+
+```
+
+kubectl port-forward --namespace kubecost deployment/kubecost-cost-analyzer 9090
```
Comment (dwbrown2) [inline _posts/2021-06-10-lens-kubecost-extension.md:54]: @AjayTripathy does this need to happen before data is available in lens?

**Review by dwbrown2 — APPROVED** [review]

**Comment by smaffulli** [comment]
adding a screenshot of the full page since the GH preview doesn't show the video
image

File: _posts/2021-06-10-lens-kubecost-extension.md (hunk 4 above)
Comment (AjayTripathy) [reply _posts/2021-06-10-lens-kubecost-extension.md:54]: yes.

Comment (AjayTripathy) [reply _posts/2021-06-10-lens-kubecost-extension.md:54]: This can be improved but it's a big lift. Let's see what people come back with for a v2

Comment (dwbrown2) [reply _posts/2021-06-10-lens-kubecost-extension.md:54]: @smaffulli this is currently inaccurate and needs to be updated

Comment (dwbrown2) [reply _posts/2021-06-10-lens-kubecost-extension.md:54]: Sorry, this just the ordering of the steps... this needs to be done before you can view the panel in the lens UI.

Comment (smaffulli) [reply _posts/2021-06-10-lens-kubecost-extension.md:54]: Fixed in the last commit

File: _posts/2021-06-10-lens-kubecost-extension.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+Once Kubecost is installed, you can add the Kubecost extension
+
+
+*   Download the [latest tar.gz file release from GitHub](https://github.com/kubecost/kubecost-lens-extension/releases). Alternatively, just copy the download URL of the release artifact.
+*   Open Lens application and select Lens menu, and then click Extensions item, or press Shift + Command + E to open the Manage Lens Extensions page.
+*   Specify the path (.tar, .tgz) or download URL to the extension package and click Install.
+
+![Install Kubecost Lens extension](/assets/images/2021-06-10-lens-kubecost/image7.png "Install Kubecost Lens extension")
+
+This will add a new panel in Lens UI showing costs of resources per namespace.
```
Comment (dwbrown2) [inline _posts/2021-06-10-lens-kubecost-extension.md:45]: Can we describe what they are actually seeing and the value provided by these figures?  Here are some example points:

Accounts for all active workloads over the measured period...
By default, pulls pricing from public cloud providers, with AWS, Azure, and GCP being supported today. You can upgrade Kubecost to reflect prices from Spot nodes, enterprise discounts, reserved instances, and more.
Custom pricing sheets for on-prem clusters

File: _posts/2021-06-10-lens-kubecost-extension.md (hunk 4 above)
Comment (dwbrown2) [reply _posts/2021-06-10-lens-kubecost-extension.md:54]: This still doesn't appear to be in the right order.

File: _posts/2021-06-10-lens-kubecost-extension.md (hunk 5 above)
Comment (dwbrown2) [reply _posts/2021-06-10-lens-kubecost-extension.md:45]: I think talking about what this integration provides would be helpful...

**Pull request merged**